ugc/ugc_tests/storage_tests.cpp

```diff
@@ -259,3 +264,42 @@ UNIT_TEST(StorageTest_LoadIndex)
   TEST(DeleteIndexFile(), ());
   TEST(DeleteUGCFile(), ());
 }
+
+UNIT_TEST(StorageTest_ContentTest)
+{
+  auto & builder = MwmBuilder::Builder();
+  m2::PointD const cafePoint(1.0, 1.0);
+  builder.Build({TestCafe(cafePoint)});
+  auto const cafeId = builder.FeatureIdForCafeAtPoint(cafePoint);
+  auto const oldUGC = MakeTestUGCUpdate(Time(chrono::hours(24 * 10)));
+  auto const newUGC = MakeTestUGCUpdate(Time(chrono::hours(24 * 300)));
+  Storage storage(builder.GetIndex());
+  storage.Load();
+  storage.SetUGCUpdate(cafeId, oldUGC);
+  storage.SetUGCUpdate(cafeId, newUGC);
+  TEST_EQUAL(storage.GetIndexesForTesting().size(), 2, ());
+  auto const toSendActual = storage.GetUGCToSend();
+
+  auto array = my::NewJSONArray();
```

Comment:
Please, keep array declaration closer to the place where it's being used.

Reply:
ok